app/controllers/concerns/idv/phone_question_ab_test_concern.rb

```diff
@@ -18,6 +18,7 @@ def maybe_redirect_for_phone_question_ab_test
       return if request.referer == idv_link_sent_url
       return if request.referer == idv_hybrid_handoff_url
       return if request.referer == idv_hybrid_handoff_url(redo: true)
+      return if request.referer == idv_cancel_url(step: 'hybrid_handoff')
```

Comment:
Wondering whether we can consolidate

Reply:
hi @dawei-nava - @soniaconnolly and I are looking to refactor our implementation here 😄